clients/imodels-client-management/src/base/internal/OperationsBase.ts

```diff
@@ -109,6 +123,9 @@ export class OperationsBase<TOptions extends OperationsBaseOptions> {
     if (params.contentType)
       headers[Constants.headers.contentType] = params.contentType;
 
+    this.addHeaders(headers, this._options.headersFactories);
```

Comment:
We should definitely have some sort of unit test that ensures this order. This order is what makes the headers passed as operation params overwrite headers passed in constructor so some test should fail if this function call order is changed